modules/rpc/handler.go

```diff
@@ -85,7 +89,7 @@ func (a authOnewayInboundMiddleware) HandleOneway(ctx context.Context, req *tran
 func authorize(ctx context.Context, host service.Host) (context.Context, error) {
 	if err := host.AuthClient().Authorize(ctx); err != nil {
 		stats.RPCAuthFailCounter.Inc(1)
-		ulog.Logger(ctx).Error(auth.ErrAuthorization, "error", err)
+		ulog.Logger(ctx).Error(auth.ErrAuthorization, zap.Error(err))
```

Comment:
Oh man, @akshayjshah, you really went above and beyond here. We have an action item to convert to unsugared usage internally. Really appreciate this!